.cbmc-batch/stubs/aws_array_list_swap_override.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * FUNCTION: aws_array_list_swap
+ *
+ * We override aws_array_list_swap because mem_swap makes CBMC
+ * struggle (because of the many memcpys) and because the
+ * array_list_get_at in before the mem_swap are unneccessary if we
+ * stub out mem_swap. Instead we add a havoc assertion on the two
```

Comment:
Where is the havoc assertion?

Reply:
I forgot to add this before. Thanks for the catch.